.github/workflows/continuous-integration.yaml

```diff
@@ -15,41 +15,50 @@ jobs:
     runs-on: '${{ matrix.os }}'
     name: 'Run Testsuite'
     steps:
+#      - name: 'Determine latest MacPorts Version'
+#        id: latest-version
+#        run: |
+#          macports_release_url=$(curl https://raw.githubusercontent.com/macports/macports-base/master/config/RELEASE_URL || curl https://trac.macports.org/export/HEAD/macports-base/config/RELEASE_URL || curl https://distfiles.macports.org/MacPorts/RELEASE_URL)
```

Comment:
Rather define two functions:

```
macports_release_url()
{
  curl https://raw.githubusercontent.com/macports/macports-base/master/config/RELEASE_URL\
  || curl https://trac.macports.org/export/HEAD/macports-base/config/RELEASE_URL\
  || curl https://distfiles.macports.org/MacPorts/RELEASE_URL
}

output_latest_version()
{
   awk -F'/v' '{ printf("latest_version=%s\n", $NF) }'
}

macports_release_url | output_latest_version >> ${GITHUB_OUTPUT}
```

Advantages: less expansions and variables, functions are easy to test/mock.

Common recommendation is to avoid ECHO with variables text, it's super brittle. Generally use `printf(1)` instead but if another tool like awk is used, why not print within awk.

Reply:
Have you considered moving his check to a separated workflow that runs periodically (daily?) and opens an issue if a new version is out?

Reply:
> Pinning the version explicitly should be the recommended approach
I strongly disagree with this, considering it's exceedinly rare for macports updates to have breaking changes.